code/modules/mob/mob_movement.dm

```diff
@@ -246,13 +246,11 @@
 		if(INCORPOREAL_MOVE_JAUNT) //Incorporeal move, but blocked by holy-watered tiles and salt piles.
 			var/turf/open/floor/stepTurf = get_step_multiz(mobloc, direct)
 			if(stepTurf)
-				var/obj/effect/decal/cleanable/food/salt/salt = locate() in stepTurf
-				if(salt)
-					to_chat(L, "<span class='warning'>[salt] bars your passage!</span>")
+				if(locate(/obj/effect/decal/cleanable/food/salt, stepTurf))
+					to_chat(L, "<span class='warning'>The salt pile bars your passage!</span>")
```

Comment:
You're reverting the recent change https://github.com/BeeStation/BeeStation-Hornet/pull/8137
Please update your branch correctly.